crates/router/src/connector/stripe/transformers.rs

```diff
@@ -276,20 +289,32 @@ pub enum BankDebitData {
 
 #[derive(Debug, Eq, PartialEq, Serialize)]
 pub struct StripeBankDebitData {
-    #[serde(rename = "payment_method_types[]")]
+    #[serde(rename = "payment_method_types[0]")]
     pub payment_method_types: StripePaymentMethodType,
     #[serde(flatten)]
     pub bank_specific_data: BankDebitData,
 }
 
+#[derive(Debug, Eq, PartialEq, Serialize)]
+pub struct StripeBankDebitType {
+    #[serde(rename = "payment_method_types[]")]
+    pub payment_method_types: StripePaymentMethodType,
+}
+#[derive(Debug, Eq, PartialEq, Serialize)]
+#[serde(untagged)]
```

Comment:
here you can use adjacent tagged enum 
https://serde.rs/enum-representations.html#adjacently-tagged

Reply:
@Narayanbhat166 changes made and pushed